### ISSUE TITLE
tide status does not inform about pending jobs

### ISSUE THREAD

**Issue by kargakis** [kubernetes/test-infra#7449]
When a PR has all the labels set but waits on one or more pending jobs, tide just reports that the PR is in the merge pool. It should pick a pending job and state that it's waiting for it to finish.

/area prow
/kind bug

@cjwagner @stevekuznetsov

**Comment by kargakis** [comment]
Still an issue.

**Comment by stevekuznetsov** [comment]
Link? Where? How?

**Comment by kargakis** [comment]
I haven't seen it work anywhere.

Example: https://github.com/openshift/origin/pull/19288

**Comment by kargakis** [comment]
Hrm, the jobs are failed and not pending. Maybe a separate issue?

**Comment by stevekuznetsov** [comment]
Hmm, @cjwagner in that linked case, we were on old `tide` and upgraded to new `tide`. We should have updated the statuses since the wanted state (`pending`) doesn't match the actual state (`successful`). So the bug is in update, not this bug specifically.

**Comment by stevekuznetsov** [comment]
@kargakis see

```
{"client":"github","component":"tide","level":"info","msg":"CreateStatus(openshift, openshift-ansible, 1d6173922d5e68b1fec7734d24ecd24b4eed7830, {pending https://deck-ci.svc.ci.openshift.org/tide.html Not mergeable. Job ci/openshift-jenkins/gcp has not succeeded. tide})","time":"2018-04-10T17:39:56Z"}
```

**Comment by stevekuznetsov** [comment]
The issue is that we would only tell you about jobs if you are LGTM+Approved and in that case you're in the pool. Issue is https://github.com/kubernetes/test-infra/issues/7279

**Comment by kargakis** [comment]
Btw, do we know what's the size limit for the status description?
